### PR TITLE
Add a smoke tests groovy - orchestrator

### PR DESCRIPTION
**Pull request by shalomnaim1**
This PR has to be reviewed by @bsquizz

### CODE REVIEW

File: Jenkinsfile-smoke.groovy [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// this 'if' statement makes sure this is a PR, so we don't run smoke tests again
+// after code has been merged into the stable branch.
+if (env.CHANGE_ID) {
+    runSmokeTest (
+        // the service-set/component for this app in e2e-deploy "buildfactory"
+        ocDeployerBuilderPath: "topological-inventory/topological-inventory-orchestrator",
+        // the service-set/component for this app in e2e-deploy "templates"
+        ocDeployerComponentPath: "topological-inventory/topological-inventory-orchestrator",
+        // the service sets to deploy into the test environment
+        ocDeployerServiceSets: "platform-mq, sources, topological-inventory",
```
Comment (bsquizz) [inline Jenkinsfile-smoke.groovy]: You'll need to remove the white space: `platform-mq,sources,topological-inventory`

Comment (shalomnaim1) [reply Jenkinsfile-smoke.groovy]: fixed

**Comment by carbonin** [comment]
I don't think we need this in every repo. @bsquizz I think one should be sufficient, right?

If we do this everywhere then we'll just end up running the same set of tests with the same set of images something like 8 times, right? I would think we would be fine if we just gate all of the PRs to `stable` on the smoke tests running on the `-api` repo.

**Comment by bsquizz** [comment]
> I don't think we need this in every repo. @bsquizz I think one should be sufficient, right?
> 
> If we do this everywhere then we'll just end up running the same set of tests with the same set of images something like 8 times, right? I would think we would be fine if we just gate all of the PRs to `stable` on the smoke tests running on the `-api` repo.

I chatted with @carbonin about this a bit more to fill him in on what we're shooting for... I think dev & QE need to talk a bit more about what and how they want to test in a smoke run, keeping in mind that the smoke tests have direct access to the pods deployed in the smoke environment, so they can do things like talk directly to kafka or a postgres DB to "seed" data or "read/validate" data. It may be the case that in some repos we can get away with only deploying/testing certain pieces of the topo-inventory set instead of deploying the whole thing repeatedly. And we may be able to seed the DB instead of using a real external provider all of the time.

**Pull request closed without merging**